contrib/graphite/blueflood.py

```diff
@@ -107,19 +122,30 @@ def make_request(self, url, payload, headers):
     return r
 
   def find_metrics_endpoint(self, endpoint, tenant):
-    return "%s/v2.0/%s/metrics/search" % (endpoint, tenant)
+    return "%s/v2.0/%s/metrics/search?include_enum_values=true" % (endpoint, tenant)
```

Comment:
will this endpoint be used to find all metrics? It's a little awkward to invoke include_enum_values for querying all metrics, assuming that all values are enums.